utils/atari_wrappers.py

```diff
@@ -0,0 +1,297 @@
+import numpy as np
+import os
+os.environ.setdefault('PATH', '')
+from collections import deque
+import gym
+from gym import spaces
+import cv2
+cv2.ocl.setUseOpenCL(False)
+
+ENV_BUMPER_AREAS = np.load('log/env_bumper_areas.npy')
```

Comment:
np.load is not stable across versions, so best to use a csv...

Reply:
I agree that all things being equal that would improve the code. Not sure if it's a top priority now.

Reply:
That's fine for now.